nilearn/maskers/tests/test_surface_maps_masker.py

```diff
@@ -0,0 +1,221 @@
+import numpy as np
+import pytest
+
+from nilearn.maskers import SurfaceMapsMasker
+from nilearn.surface import SurfaceImage
+
```

Comment:
Can you add test to check we are compliant with sklearn ?

see https://github.com/nilearn/nilearn/blob/6a2107674849cb19d6916cba6793c07a6f483b45/nilearn/maskers/tests/test_surface_masker.py#L11-L43